apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/ExperimentsResource.java

```diff
@@ -132,6 +134,21 @@ public Response create(
         return Response.created(uri).build();
     }
 
+    @POST
+    @Path("/delete")
+    @Operation(operationId = "deleteExperimentsById", summary = "Delete experiments by id", description = "Delete experiments by id", responses = {
```

Comment:
@thiagohora will this verify that the user has permissions for this workspace ? 
or we just take the workspaceId and add that to the query in the DB ?

Reply:
All endpoints in `/v1/private/**` are authenticated and authorized against the workspace. Once that's done, a request context bean will be set with the currently authorized workspace ID.  Then, all queries are read from the context.

Reply:
But it was a good point, I added tests for it.